src/sugar3/speech.py

```diff
@@ -278,7 +278,12 @@ def make_pipeline(self, command):
             self.stop_sound_device()
             del self._pipeline
 
-        self._pipeline = Gst.parse_launch(command)
+        try:
+            self._pipeline = Gst.parse_launch(command)
+        except:
```

Comment:
@SAMdroid-apps what about:

except Exception as error:
\t logging.error('The spech plugin can't be found, %s' %  error)